python/python/bystro/api/search.py

```diff
@@ -0,0 +1,264 @@
+from typing import Optional, Mapping, Any, Union, Collection
+from requests.auth import AuthBase
+
+from opensearchpy import OpenSearch, RequestsHttpConnection, AsyncOpenSearch, AsyncHttpConnection
+
+from bystro.api.auth import CachedAuth
+
+QUERY_ENDPONIT = "/api/jobs/{job_id}/opensearch"
```

Comment:
Is this a typo? It matches line 219 and 257 but it seems like it should be QUERY_ENDPOINT?

Reply:
hah yeah